client/src/main/java/io/pravega/client/KeyValueTableFactory.java

```diff
@@ -52,19 +51,12 @@ static KeyValueTableFactory withScope(String scope, ClientConfig config) {
      * Creates a new {@link KeyValueTable} that provides access to a Pravega Key-Value Table.
      *
      * @param keyValueTableName   Name of the {@link KeyValueTable}.
-     * @param keySerializer       A {@link Serializer} for {@link KeyValueTable} Keys. Refer to the {@link KeyValueTable}
-     *                            Javadoc for constraints relating to the size of the serialization.
-     * @param valueSerializer     A {@link Serializer} for {@link KeyValueTable} Values. Refer to the {@link KeyValueTable}
-     *                            Javadoc for constraints relating to the size of the serialization.
      * @param clientConfiguration A {@link KeyValueTableClientConfiguration} to use for configuring the
      *                            {@link KeyValueTable} client.
-     * @param <KeyT>              Key Type.
-     * @param <ValueT>            Value Type.
      * @return A {@link KeyValueTable} that provides access to the requested Key-Value Table.
      */
-    <KeyT, ValueT> KeyValueTable<KeyT, ValueT> forKeyValueTable(
-            @NonNull String keyValueTableName, @NonNull Serializer<KeyT> keySerializer,
-            @NonNull Serializer<ValueT> valueSerializer, @NonNull KeyValueTableClientConfiguration clientConfiguration);
+    KeyValueTable forKeyValueTable(
```

Comment:
If I remember when we came up with that name last year, we had to choose between `createKeyValueTable` or something else. The problem I had with `create` was that it may be confusing to some folks who may believe we are actually creating the table at that point. This method simply creates a client-side object that provides access to a KVT.

I looked at our EventStreamWriter/Reader and for that case, using `create` makes a lot of sense - we are creating a "writer" or a "reader". In the KVT case, however we don't have such a denomination.

If you have a better suggestion than `for` (which is not `create`), then we can definitely address this.

Reply:
I think the problem is implied by the name of this interface. 
All of the others are: "Batch**Client**Factory", "ByteStream**Client**Factory", "Synchronizer**Client**Factory"
So when they have methods with "create" such as "createRevisionedStreamClient" the meaning and implication is clear. 

If this were a "KeyValueTableClientFactory" and it had a method "createKeyValueTableClient" there would be no confusion. It is rather long. We could try to shorten it by either dropping "table" or abbreviating "KeyValue" as "KV". I don't have a strong opinion either way.

Reply:
I don't have a strong preference either. I created #5978 to track this and get it done later - I don't want to delay this PR over this pre-existing condition.